src/main/java/ua/kpi/iasa/IASA_Organiser/controller/Controller.java

```diff
@@ -34,77 +44,187 @@ public void init(View view) {
+     *
+     * @param event
+     * @see EventService
+     */
     public void removeEvent(Event event) {
         logger.debug("Method was called with {}", event);
         final EventService currentEventService = getEventService();
         currentEventService.removeEvent(event);
     }
 
+
+    /**
+     * This method gets all events {@link Event} with given date field from the dataStorage
+     * <p>
+     * Receives instance of LocalDate, returns result of calling {@link CalendarService#findEventsByDate(LocalDate)}
+     * with received instance.
+     *
+     * @param searchDate
+     * @return list of events which have given date field
+     * @see CalendarService
+     */
     public List<Event> findEventsByDate(LocalDate searchDate) {
         logger.debug("Method was called with {}", searchDate);
         final CalendarService currentCalendarService = getCalendarService();
         return currentCalendarService.findEventsByDate(searchDate);
     }
 
+    /**
+     * This method gets all events {@link Event} with given tag field from the dataStorage
+     * <p>
+     * Receives instance of Tag, returns result of calling {@link CalendarService#findEventsByTag(Tag)}
+     * with received instance
+     *
+     * @param searchTag
+     * @return list of events which have given tag field
+     * @see CalendarService
+     */
     public List<Event> findEventByTag(Tag searchTag) {
         logger.debug("Method was called with {}", searchTag);
         final CalendarService currentCalendarService = getCalendarService();
         return currentCalendarService.findEventsByTag(searchTag);
     }
 
+    /**
+     * This method gets all events {@link Event} with given priority field from the dataStorage
+     * <p>
+     * Receives instance of Priority, returns result of calling {@link CalendarService#findEventsByPriority(Priority)}
+     * with received instance
+     *
+     * @param searchPriority
+     * @return list of events with given priority field
+     * @see CalendarService
+     */
     public List<Event> findEventByPriority(Priority searchPriority) {
         logger.debug("Method was called with {}", searchPriority);
         final CalendarService currentCalendarService = getCalendarService();
         return currentCalendarService.findEventsByPriority(searchPriority);
     }
 
+    /**
+     * This method sorts all existing events {@link Event} by priority
+     * <p>
+     * Calls {@link CalendarService#sortEventsByPriority()} and returns sorted list of events
+     *
+     * @return list of events sorted by priority
+     */
     public List<Event> sortEventsByPriority() {
         logger.debug("Method was called...");
         final CalendarService currentCalendarService = getCalendarService();
         return currentCalendarService.sortEventsByPriority();
     }
 
+    /**
+     * This method gets instance of EventService{@link EventService}
+     *
+     * @return instance of EventService
+     * @see EventService
+     */
```

Comment:
Redundant JavaDocs;
Don't write JavaDocs on setter or getters;
Put that method on the end of class